### ISSUE TITLE
[Bug] no preview items

### ISSUE THREAD

**Issue by mpas97** [lucaro/DRES#107]
It can't load the preview images. The requests are returning `429 (Too Many Requests)`.
<img width="1674" alt="grafik" src="https://user-images.githubusercontent.com/9994218/84254965-3aa6ea80-ab12-11ea-9007-926c8fa5c99a.png">

**Comment by lucaro** [comment]
A 429 is the intended behavior when the backend is not able to serve the requested preview images quickly enough, in order to manage the load. You not having any preview images points to a problem with your local setup. Are the relevant video files readable from your instance and is the external ffmpeg dependency executable?

**Comment by sauterl** [comment]
I cannot reproduce this issue locally, you might want to double-check your setup.

**Comment by mpas97** [comment]
@lucaro If the load would be a problem, yes, but I am the only one using this instance and as I can't do submissions, judging and having a look on the viewer in parallel I don't think there is any (havy) load at all. 
@silvanheller do you know more regarding ffmpeg dependency etc?

**Comment by lucaro** [comment]
The 'load' is estimated by the number of pending images to be generated. If the generation mechanism does not return images at all, which appears to be the case in your setup, this estimated load only increases. Whenever the mechanism does not return the requested image within the specified timeout, the backend assumes increased load and returns a 429. So this is expected behavior in this case.

**Comment by silvanheller** [comment]
Yeah, avs preview images are an issue on our setup which I haven't had the time to investigate yet. I'll have a look and see if I can report a specific issue. It's good to know that 429 is expected behavior.
